tequilapi/endpoints/nat.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package endpoints
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysteriumnetwork/node/nat"
+	"github.com/mysteriumnetwork/node/nat/natevents"
+	"github.com/mysteriumnetwork/node/tequilapi/utils"
+)
+
+const (
+	statusNotFinished = "not_finished"
+	statusSuccessful  = "successful"
+	statusFailure     = "failure"
+)
+
+// NATStatusDTO gives information about NAT traversal success or failure
+// swagger:model NATStatusDTO
+type NATStatusDTO struct {
+	Status string  `json:"status"`
+	Error  *string `json:"error,omitempty"`
```

Comment:
`omitempty` should work for empty string too, no need it to be a pointer. 
```suggestion
	Error  string `json:"error,omitempty"`
```